x-pack/plugins/security_solution/public/flyout/document_details/isolate_host/header.test.tsx

```diff
@@ -39,4 +49,36 @@ describe('<PanelHeader />', () => {
     expect(getByTestId(FLYOUT_HEADER_TITLE_TEST_ID)).toBeInTheDocument();
     expect(getByTestId(FLYOUT_HEADER_TITLE_TEST_ID)).toHaveTextContent('Release host');
   });
+
+  it.each(['isolateHost', 'unisolateHost'])(
+    'should display beta badge on %s host message for SentinelOne alerts',
+    (action) => {
+      (useIsolateHostPanelContext as jest.Mock).mockReturnValue({
+        isolateAction: action,
+      });
+      mockUseIsExperimentalFeatureEnabled.mockReturnValue(true);
+      mockIsAlertFromSentinelOneEvent.mockReturnValue(true);
+
+      const { getByTestId } = renderPanelHeader();
+
+      expect(getByTestId(FLYOUT_HEADER_TITLE_TEST_ID)).toBeInTheDocument();
+      expect(getByTestId(FLYOUT_HEADER_TITLE_TEST_ID)).toHaveTextContent('Beta');
+    }
+  );
+
+  it.each(['isolateHost', 'unisolateHost'])(
```

Comment:
Requesting you to add one more test to test impact of `experimental_flag = false` on the visibility of the `beta` badge.